[ADD] timesheets: billing rate leaderboad (#13921)
odoo/documentation: content/applications/services/timesheets/overview/billing_rates.rst

.. _billing_rates/targets:

Billing rate indicators
=======================

Configuration
-------------

To enable billing rate indicators, navigate to :menuselection:`Timesheets --> Configuration --> Settings`,
then enable :guilabel:`Billing Rate Indicators`, and press :guilabel:`Save`.

A new item, :guilabel:`Billing Time Targets` is then added in the :guilabel:`Configuration` menu. To
set up the targets, click :guilabel:`Billing Time Targets`, then encode the :guilabel:`Monthly
Billing Time Target` in hours or days, depending on the **Encoding Method** selected in the
**Timesheets** app configuration.

.. note::
   Once the :guilabel:`billing rate indicators` have been enabled in **Timesheets** app
   configuration, you can also navigate to the **Employees** app, access the employee form that you
   wish to edit, and navigate to the :guilabel:`Settings` tab. Scroll down to
   :guilabel:`Application settings`, then encode the :guilabel:`Monthly Billing Time Target`.

Using the billing rate indicators
---------------------------------

Once enabled, the billing rate indicators are displayed in the upper right corner of the

> :guilabel:`My Timesheets` in Kanban, grid, and list views for all users. The monthly amount of time

Author: A word WAS missing, I don't know how I did that! I actually also had a hard time coming up with what to call it, I remember asking Dallas (you weren't there, TRAGICALLY), and we went with "menu". But we can definitely discuss together if it doesn't work for you!

red if the user falls below the target and in green once the target is reached.

Below the billing rate indicators, the total monthly amount of time logged by the user is also
displayed.

.. image:: billing_rates/indicators.png
   :alt: Billing rate indicators.

.. _billing_rates/leaderboard:

Billing rate leaderboard
========================

Configuration
-------------

After the **monthly billing time targets** have been enabled, Odoo offers the possibility of
activating a **billing rate leaderboard** in order to motivate employees and enhance workplace
transparency.

To enable the billing rate leaderboard, navigate to :menuselection:`Timesheets --> Configuration --> Settings`,
activate :guilabel:`Billing Rate Leaderboard`, then click :guilabel:`Save`.

Using the billing rate leaderboard
----------------------------------

The billing rate leaderboard is then displayed in the upper right corner of the
:guilabel:`My Timesheets` view, next to the billing rate indicators. It displays the current top
three performers who have logged the highest percentage of their allocated billing hours.

> It also shows the amount of the time logged by the signed in user in the format: logged billable
> time / billable time target, as well as the total time logged.

Reviewer: ```suggestion
It also shows the amount of time logged by the signed-in user in the format: logged billable
time / billable time target, as well as the total time logged.
```

Author: THE AMOUNT OF THE TIME hahahaha I love it, I wish we could keep it! (but it's fixed now, rip)

Clicking on the area of the top three performers opens the leaderboard for all team members. Use the
drop-down menu in the upper left corner to switch between the :guilabel:`Billing Rate Leaderboard`
and the :guilabel:`Total Time Leaderboard`, which displays the total time logged by team members
across billable and internal projects.

.. image:: billing_rates/leaderboard.png
   :alt: Billing rate leaderboard.

.. _billing_rates/tips:

Leaderboard tips
================

**Daily motivational tips**, displayed on the right side of the leaderboard, enhance the billing
rate and total time leaderboard. The tips are randomly selected and change daily.

> To create or edit existing tips, navigate to :menuselection:`Timesheets --> Configuration --> Tips`.
> To create a new tip, click :guilabel:`New`, and to edit the existing one, double-click on its text.

Reviewer: I feel like maybe we could group these two sentences (it would also avoid repetitions):
```suggestion
To create or edit existing tips, navigate to :menuselection:`Timesheets --> Configuration --> Tips`, then click :guilabel:`New` or double-click a tip's text.
```
